pypiserver/core.py

```diff
@@ -92,8 +92,12 @@ def auth_by_htpasswd_file(htPsswdFile, username, password):
 
 
 def auth_by_pam(username, password):
-    import pam
-    return pam.authenticate(username, password)
+    try:
+        import pam
+        return pam.authenticate(username, password)
+    except ImportError as error:
+        log.error('PAM module not found. Please install pam module')
+        return False
 
```

Comment:
Sorry :-( 
We don't `try/catch` for missing dependencies - the original exception is quite explanatory to the user (in combination with the  documentation, which is the one needing improvement).